query-node/mappings/src/workingGroups.ts

```diff
@@ -1000,3 +1001,25 @@ export async function workingGroups_BudgetSpending({ store, event }: EventContex
 
   await store.save<WorkingGroup>(group)
 }
+
+export async function workingGroups_WorkingGroupBudgetFunded({
+  store,
+  event,
+}: EventContext & StoreContext): Promise<void> {
+  const [memberId, amount, rationale] = new WorkingGroups.WorkingGroupBudgetFundedEvent(event).params
+  const group = await getWorkingGroup(store, event)
+
+  const budgetFundedEvent = new BudgetFundedEvent({
+    ...genericEventFields(event),
+    group,
+    member: new Membership({ id: memberId.toString() }),
+    amount,
+    rationale: bytesToString(rationale),
+  })
+
+  await store.save<BudgetFundedEvent>(budgetFundedEvent)
+
+  group.budget = new BN(group.budget).add(amount)
```

Comment:
I think `group.budget` is already a `BN`. So any specific reason for reconstructing the type.
```suggestion
  group.budget = group.budget.add(amount)
```